cmd/cni-metrics-helper/main.go

```diff
@@ -85,6 +105,17 @@ func main() {
 		}
 	}
 
+	prometheusENV, found := os.LookupEnv(envEnablePrometheusMetrics)
+	if found {
+		if strings.Compare(prometheusENV, "yes") == 0 || strings.Compare(prometheusENV, "true") == 0 {
```

Comment:
nit: should we do case-insensitive compare here? i.e. convert `prometheusENV` using `lower`

Reply:
I just kept is same as CW..maybe I can change in both places..